services/httperrors/httperrors_test.go

```diff
@@ -44,3 +51,24 @@ func TestWrite(t *testing.T) {
 		t.Error("the body should contains the jsonified HTTPError")
 	}
 }
+
+func TestNewErrorNotFound(t *testing.T) {
```

Comment:
TestNewErrorNotFound, TestNewInternalServerError, TestNewUnauthorizedError not relevant test, in this case we want to check if default value is well set